qt_gui_app/scripts/qt_gui_app

```diff
@@ -3,13 +3,10 @@
 import os
 import sys
 
-import rospkg
+from ament_index_python.resources import get_resource
```

Comment:
The package needs to declare an exec depend on `ament_index_python` to ensure it is available.